e2e/basic-example/basic-example.js

```diff
@@ -33,7 +33,24 @@ describe('the Basic example page', function () {
 
   describe('the dragging and dropping of nodes', function () {
     it('should allow moving a node below another node', function () {
-      // TODO
+      var handles = basicExamplePage
+        .rootNodes.get(1)
+        .all(by.repeater('node in node.nodes'))
+        .all(by.css('[ui-tree-handle]'));
+
+      handles.get(0).getText().then(function(nodeTextBeforeDrag) {
+
+        browser.actions().dragAndDrop(
+          handles.get(0),
+          { x: 0, y: 200 }
+        ).perform();
+
+        handles.get(0).getText().then(function(nodeTextAfterDrag) {
+          console.log(nodeTextBeforeDrag, nodeTextAfterDrag);
+          expect(nodeTextBeforeDrag).not.toBe(nodeTextAfterDrag);
```

Comment:
Instead of checking if the label does not equal the original label, I would explicitly check the values of both labels here. That would result in 2 expects:

**Before the move**
```
expect(firstNodeLabel).toBe('node1');`
expect(secondNodeLabel).toBe('node2');`
```

**After moving the node**
```
expect(firstNodeLabel).toBe('node2');
expect(secondNodeLabel).toBe('node1');
```